src/wavm/openmp.cpp

```diff
@@ -1,22 +1,27 @@
-#include "wavm/openmp/openmp.h"
```

Comment:
This file contains the OpenMP host interface functions, so is the single entrypoint to all the OpenMP code